aut_docs/Installation_Setup.md

```diff
@@ -130,10 +130,14 @@
     ```
         \q
     ```
+- Change directory to VMS having the ``manage.py`` in it
+    ```bash
+      cd vms/
+    ```
 
-- Use the secret_key by running this command
+- Use the following command to handle the environment variables (secret key, DEBUG, email keys, db settings)
```

Comment:
Please remove this from the topic of Setup PostgreSQL. You can put this in the topic of one-time setup because it need only one time setup.

Reply:
> Please remove this from the topic of Setup PostgreSQL. You can put this in the topic of one-time setup because it need only one time setup.

@yashshah2820